modules/test/playwright/tests/journal-web/journalArticleAutosave.spec.ts

```diff
@@ -95,6 +98,73 @@ autoSaveTest(
 	}
 );
 
+autoSaveTest(
+	'Default Language can be changed when editing default values',
+	{
+		tag: '@LPD-38269',
+	},
+	async ({
+		apiHelpers,
+		journalEditStructureDefaultValuesPage,
+		page,
+		site,
+		systemSettingsPage,
+	}) => {
+		await systemSettingsPage.goToSystemSetting(
+			'Web Content',
+			'Administration'
+		);
+
+		await page.getByLabel('Changeable Default Language').check();
+
+		const updateButton = page.getByRole('button', {
+			name: 'Update',
+		});
+
+		const saveButton = page.getByRole('button', {
+			name: 'Save',
+		});
+
```

Comment:
We didn't need it now, I think this is the SF error

Reply:
Ohh sorry I missed those lines.